.gitignore

```diff
@@ -3,3 +3,4 @@ coverage/
 test/coverage/
 .bundle
 bundle
+.tags
```

Comment:
I can rip it out...It is generated by whatever some atom plugin is kicking of ctags with. I just added it to my global .gitignore.